src/assets/specs/README.md

```diff
@@ -0,0 +1,42 @@
+## Generators
+
+### To generate go code:
+
+oapi-codegen src/assets/specs/command-api.yaml > api/commands.go
```

Comment:
Please use backticks quote these.

Should this happen automatically (e.g. on `npm install`), similar what we do with the moby-openapi specs?

Should we convert the moby stuff to use this generator instead?

Reply:
Not yet. But I'm big on not putting generated code in revision control, meaning that yes, `npm run postinstall` is the right place to generate it. Maybe we should start using `make` more often (or some other build system) so that we only build things that have changed.

Reply:
Yeah, I suspect we might want something that manages dependencies like `make`. Ideally, though, we'd want something that we can install via `npm`, so that it runs on Windows too without installing additional tooling…